### PR TITLE
Add support for OTP 21

### PR DESCRIPTION
**Pull request by urbanserj**
JIRA: https://jira.mesosphere.com/browse/DCOS_OSS-3655

### CODE REVIEW

**Comment by urbanserj** [comment]
@GoelDeepak request for review

File: apps/dcos_l4lb/test/dcos_l4lb_lashup_vip_listener_SUITE.erl [hunk 1]
```diff
@@ -70,6 +70,7 @@ lookup_failure3(Config) ->
 
 lookup_vip(_Config) ->
   {ok, _} = update_de8b9dc86_marathon(),
+  timer:sleep(100),
```
Comment (GoelDeepak) [inline apps/dcos_l4lb/test/dcos_l4lb_lashup_vip_listener_SUITE.erl]: does it that long?

Comment (urbanserj) [reply apps/dcos_l4lb/test/dcos_l4lb_lashup_vip_listener_SUITE.erl]: Well, it's hard to say. We don't have metric for this. It's a flaky test. What value do you think we should use here?

Comment (GoelDeepak) [reply apps/dcos_l4lb/test/dcos_l4lb_lashup_vip_listener_SUITE.erl]: Sorry, I was under impression that the input to `sleep` is in seconds. Looks like it's in milliseconds. We should be fine then

**Pull request merged**